test/python/transpiler/test_time_unit_conversion.py

```diff
@@ -0,0 +1,90 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test the optimize-1q-gate pass"""
+
+import itertools
+from ddt import ddt, idata, unpack
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import Delay
+from qiskit.circuit.library import XGate
+from qiskit.transpiler import InstructionDurations, InstructionProperties, Target
+from qiskit.transpiler.passes import TimeUnitConversion
+from qiskit.converters import circuit_to_dag
+from qiskit.transpiler.exceptions import TranspilerError
+from test import QiskitTestCase  # pylint: disable=wrong-import-order
+
+
+@ddt
+class TestTimeUnitConversion(QiskitTestCase):
```

Comment:
Personally, I like having separate files for different transpiler tests, but in this case, I think it'd be best to place your new tests in `test_transpiler.py` beside the others. And, I'd just inline all of the target setup stuff (as you'll see, we do that for the existing tests), since it's fairly specific for what we're testing.

Reply:
Tests moved to test_transpiler in [b749029](https://github.com/Qiskit/qiskit/pull/14027/commits/b7490299ec032dc501770e0a912175a89c20a90f).